test/CMakeLists.txt

```diff
@@ -6,21 +6,23 @@ include(chimeraFunctions)
 find_package(Boost REQUIRED COMPONENTS python)
 find_package(PythonLibs REQUIRED)
 
-include_directories(SYSTEM
-  ${Boost_INCLUDE_DIRS}
-  ${PYTHON_INCLUDE_DIRS}
-)
-
 add_chimera_binding(
   EXCLUDE_FROM_ALL
   TARGET test_binding
   SOURCES dart-example.cpp
   NAMESPACES dart
 )
+
+target_include_directories(test_binding SYSTEM
+  PRIVATE ${Boost_INCLUDE_DIRS}
+  PRIVATE ${PYTHON_INCLUDE_DIRS}
+)
+
 target_link_libraries(test_binding
   ${Boost_PYTHON_LIBRARIES}
```

Comment:
I didn't test for Boost.Python, but it seems the pattern of boost component variable is `Boost_COMPONENTNAME_LIBRARY`. So this might need to be changed to `Boost_PYTHON_LIBRARY`. Please ignore if I'm wrong, but I confirmed this for [Boost.FileSystem](https://github.com/personalrobotics/aikido/pull/154/files#diff-768ce6c21087a82c0c1a47637a8d8a18R56).